### PR TITLE
SYS-3560 Tidy up and new ocw locking tests

### PR DESCRIPTION
**Pull request by nahuseyoum**
This PR does a few things:
 - Removes any legacy code that is throwing a warning when compiling
 - Fixing existing tests broken specifically by the ocw locking changes
 - Adds new ocw lock tests for the summary pallet

### CODE REVIEW

**Review by ivan-cholakov — APPROVED** [review]

File: pallets/summary/src/tests/test_ocw_locks.rs [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+        slot_number,
+        slot_validator,
+        other_validator,
+        block_number_for_next_slot,
+        grace_period,
+        summary_last_block_in_range,
+    }
+}
+
+
```
Comment (fluorostani) [inline pallets/summary/src/tests/test_ocw_locks.rs:72]:
```suggestion
```

Comment (fluorostani) [inline pallets/summary/src/tests/test_ocw_locks.rs:72]:
```suggestion
```

Comment (fluorostani) [inline pallets/summary/src/tests/test_ocw_locks.rs:72]: Remove new lines

File: pallets/summary/src/tests/test_ocw_locks.rs [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+            Summary::offchain_worker(context.block_number_for_next_slot + 2);
+            assert_eq!(false, pool_state.read().transactions.is_empty());
+        });
+    }
+}
+
+mod record_summary_locks {
+    use super::*;
+
+
```
Comment (fluorostani) [inline pallets/summary/src/tests/test_ocw_locks.rs:141]: Remove new line

**Review by fluorostani — APPROVED** [review]
Looks good but you might want to run the formatter

**Review by fluorostani — APPROVED** [review]

**Pull request merged**